src/components/form-elements/draft-js-mention-selector/DraftJSMentionSelector.js

```diff
@@ -20,12 +22,30 @@ import Toggle from '../../toggle/Toggle';
 const mentionStrategy = (contentBlock, callback, contentState) => {
     contentBlock.findEntityRanges(character => {
         const entityKey = character.getEntity();
-
         const ret = entityKey !== null && contentState.getEntity(entityKey).getType() === 'MENTION';
         return ret;
     }, callback);
 };
 
+/**
+ * Scans a Draft ContentBlock for timestamp entity ranges
+ * @see docs at {@link https://draftjs.org/docs/advanced-topics-decorators.html#compositedecorator}
+ * @param {ContentBlock} contentBlock
+ * @param {function} callback
+ * @param {ContentState} contentState
+ */
+const timestampStrategy = (contentBlock: any, callback: (start: number, end: number) => void, contentState: any) => {
+    if (!contentBlock || !contentState) return;
```

Comment:
agree with Jackie that the braces improve readability
```suggestion
    if (!contentBlock || !contentState) {
        return;
    }
```